### PR TITLE
Add a `--version` cmdline option to test-tool.

### PR DESCRIPTION
**Pull request by fabiotudone-da**
CHANGELOG_BEGIN
[TestTool] Add a `--version` command line option to test tool.
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by fabiotudone-da** [comment]
@stefanobaghino-da It didn't actually compile even though the IDE though it was all fine, the dependency was not actually there. Duplicated that small snippet and also adjusted the build instead, it would be nice to share that code though.

**Comment by stefanobaghino-da** [comment]
Don't add that dependency, please, it's in the sandbox package and the test tool should not depend on the sandbox. I'm refactoring that code away from the sandbox, wait for that to be in to move on.

**Comment by stefanobaghino-da** [comment]
Ok, I see what you have done. I'm already working to fix this, please wait for that to be in.

**Comment by fabiotudone-da** [comment]
> Don't add that dependency, please, it's in the sandbox package and the test tool should not depend on the sandbox. I'm refactoring that code away from the sandbox, wait for that to be in to move on.

Not added, copied `BuildInfo` for now.

**Comment by stefanobaghino-da** [comment]
I t would be the fourth copy of that code in the repository, two of which don't even close the stream.

**Comment by fabiotudone-da** [comment]
> It would be the fourth copy of that code in the repository, two of which don't even close the stream.

It's OK to wait for it to be factored, I can rebase as soon as there's an unified version.

**Comment by stefanobaghino-da** [comment]
> > Don't add that dependency, please, it's in the sandbox package and the test tool should not depend on the sandbox. I'm refactoring that code away from the sandbox, wait for that to be in to move on.
> 
> Not added, copied `BuildInfo` for now.

Yes, that's why I replied that I've seen what you have done. But as I mentioned it would be the fourth copy of the same behavior in the repository. I'm already working on this.

**Comment by stefanobaghino-da** [comment]
https://github.com/digital-asset/daml/pull/5004

**Comment by stefanobaghino-da** [comment]
#5004 has been merged.

File: ledger/ledger-api-test-tool/BUILD.bazel [hunk 1]
```diff
@@ -28,6 +28,10 @@ load(
     ":conformance.bzl",
     "conformance_test",
 )
+load(
+    "@build_environment//:configuration.bzl",
+    "mvn_version",
+)
```
Comment (stefanobaghino-da) [inline ledger/ledger-api-test-tool/BUILD.bazel]: I don't think this is needed anymore.

File: ledger/ledger-api-test-tool/BUILD.bazel [hunk 2]
```diff
@@ -108,6 +112,8 @@ da_scala_binary(
         "src/main/resources/logback.xml",
         ":PackageManagementTest",
         "//ledger/test-common:dar-files",
+    ] + [
+        "//:MVN_VERSION",
```
Comment (stefanobaghino-da) [inline ledger/ledger-api-test-tool/BUILD.bazel]: This can probably be dropped now, thanks.

Comment (fabiotudone-da) [reply ledger/ledger-api-test-tool/BUILD.bazel]: Oh, i guess it is brought in by the dependency. OK

File: ledger/ledger-api-test-tool/BUILD.bazel (hunk 1 above)
Comment (stefanobaghino-da) [reply ledger/ledger-api-test-tool/BUILD.bazel]: Does this have to stay here?

Comment (fabiotudone-da) [reply ledger/ledger-api-test-tool/BUILD.bazel]: Sorry, missed it earlier, tried without it and it's indeed not needed.

**Pull request merged**